core/src/test/java/io/parsingdata/metal/AutoEqualityTest.java

```diff
@@ -319,6 +321,9 @@ private static Object[] generateObjectArrays(final Class<?> c) throws IllegalAcc
         cons.setAccessible(true);
         List<List<Supplier<Object>>> args = new ArrayList<>();
         for (Class<?> cl : cons.getParameterTypes()) {
+            if (!mapping.containsKey(cl)) {
+                throw new AssertionError("Please add a mapping for type " + cl.getSimpleName());
+            }
```

Comment:
Added this error for our own convenience to save time in case of an error. If missing, the test somewhere upstream will cause a NullPointerException and we have to debug it to figure out it already went wrong here.